quesma/ingest/parser.go

```diff
@@ -114,10 +114,7 @@ func SchemaToColumns(schemaMapping *schema.Schema, nameFormatter TableColumNameF
 
 	for _, field := range schemaMapping.Fields {
 		var fType string
-
-		// We should never have dots in the field names, see 4 ADR
-		internalPropertyName := strings.Replace(field.InternalPropertyName.AsString(), ".", "::", -1)
-
+		internalPropertyName := util.FieldToColumnEncoder(field.InternalPropertyName.AsString())
```

Comment:
as chatted on call, we could've used `schemaMapping *schema.Schema` object instead of function here

Reply:
For now `schema.Schema` does not store info about field encodings. They are stored in `schema.Registry`. I will add comment here, but it would be cleaner and safer to do that in subsequent PR